backend/functions/src/utils/on-create-document.ts

```diff
@@ -0,0 +1,4 @@
+export function onCreateDocument(obj: object) {
```

Comment:
`object` is a vary vague type. `object` could be almost anything. A better type to use here is `Record<string, unknown>` - this means the argument is an object, where the value types are `unknown`.